docs/Developers/Git_Basics.rst

```diff
@@ -287,6 +287,13 @@ This will fetch the commits from the remote repository and will merge it into
 the branch where you are currently working, and move all of the local commits
 that are ahead of the rebased branch to the top of the history on that branch.
 
+Mid-Rebase Conflicts
```

Comment:
I feel like this section should be under the Common Git Issues section instead of here.